digits/scheduler.py

```diff
@@ -175,6 +177,30 @@ def add_job(self, job):
             return False
         else:
             self.jobs.append(job)
+
+            if flask._app_ctx_stack.top is not None:
```

Comment:
@lukeyeager, as far as imports go, the only import change from origin/master in webapp or scheduler is in scheduler.py

```from digits.utils import subclass, override```